### ISSUE TITLE
How about thumbnail crop position ?

### ISSUE THREAD

**Issue by rambuvn** [MatthewRuddy/Wordpress-Timthumb-alternative#16]
Timbthumb have option to choose where will be crop. Do you have any solution for that

**Comment by gulariav** [comment]
Yup Need to crop image from top instead of middle. Is any one having luck?

**Comment by samuelguebo** [comment]
Found my a way out here https://github.com/mendezcode/mr-image-resize/blob/master/mr-image-resize.php line #53 - line #99. mr-image-resize is a script based on Mathew Ruddy's. It deals nicely with cropping option just like timthumb used to.
